nodes/x86_64_ubuntu_16_04_LTS.json

```diff
@@ -0,0 +1,18 @@
+{
+  "contact": {
+    "name": "Andrew Breckenridge",
+    "email": "swiftci@fb.com",
+    "company": "Facebook, Inc."
+  },
+  "node": {
+      "platform": "Android",
+      "os_version": "Ubuntu 16.04 LTS"
+  },
+  "jobs": [
+    {
+      "display_name": "Swift - Android (Tools RA, Stdlib RD) (master))",
```

Comment:
I think you've got one too many `)`'s here, too.

Reply:
Thanks @harlanhaskins, I must have been really excited when I saw this was released

Reply:
I don’t blame you!